docs/basics/steps/3_Cast_and_Challenge.md

```diff
@@ -2,4 +2,9 @@
 
 Each ballot that is completed by a voter must be either cast or challenged. A cast ballot is a ballot that the voter accepts as valid and wishes to include in the official election tally. A challenge ballot, also referred to as a spoiled ballot, is a ballot that the voter does not accept as valid and wishes to exclude from the official election tally.
 
-ElectionGuard includes a mechanism to mark a specific ballot as either cast or spoiled. Cast ballots are included in the tally record, while spoiled ballots are not. Spoiled ballots are decrypted into plaintext and published along with the rest of the election record.
\ No newline at end of file
+![Encrypt][encrypt-image]
+
+<!-- Links -->
```

Comment:
Please put the links at the bottom of the page. I'm gonna have @rc_ms review this for content but this looks like a good start.

Reply:
Okay, I will do that as soon as possible! Thanks for your feedback.

Reply:
Well, @keithrfung I put the links at the bottom of the page. Let me know if this is fine.

Reply:
Awesome this looks good. @rc-ms will review the content. Probably on Monday but thanks for your contribution.